src/components/EditProfilePage.vue

```diff
@@ -0,0 +1,78 @@
+<template>
+  <div class="vue-template user-form">
+    <form>
+      <h3>Edit User Profile</h3>
+
+      <div class="form-group">
+        <label>First Name</label>
+        <input type="text" @input="firstname($event.target.value)" class="form-control form-control-lg"/>
```

Comment:
Beter use two-way data binding `v-model="firstName"` instead of `@input="firstname($event.target.value)"`

Reply:
done